vsintegration/src/FSharp.Editor/Completion/CompletionProvider.fs

```diff
@@ -218,8 +219,7 @@ type internal FSharpCompletionProvider
         asyncMaybe {
             let document = context.Document
             let! sourceText = context.Document.GetTextAsync(context.CancellationToken)
-            let defines = projectInfoManager.GetCompilationDefinesForEditingDocument(document)
-            do! Option.guard (CompletionUtils.shouldProvideCompletion(document.Id, document.FilePath, defines, sourceText, context.Position))
+            do! Option.guard (this.ShouldTriggerCompletion(sourceText, context.Position, context.Trigger, null))
```

Comment:
ShouldTriggerCompletion will fire once you type a character; however, you can ctrl+space to bring up completions and this will not fire ShouldTriggerCompletion. Instead, it will go right to ProvideCompletionsAsync.

ProvideCompletionsAsync should not be checking for a dot to get completions. That's the responsibility of ShouldTriggerCompletions.

I recommend making go back to what it was.